addon/components/code-inline.js

```diff
@@ -1,3 +1,5 @@
+/* global Prism */
```

Comment:
Ah, ok. Awesome. That's great information thanks @justin-lau for your work on this.